dashboard/app/templates.html

```diff
@@ -283,3 +283,44 @@ <h1><a href="/{{$.Namespace}}">syzbot</a></h1>
 	{{end}}
 {{end}}
 {{end}}
+
+{{/* Show crashes */}}
+{{define "crashes"}}
+{{if .}}
+		<thead>
+		<tr>
+			<th><a onclick="return sortTable(this, 'Manager', textSort)" href="#">Manager</a></th>
+			<th><a onclick="return sortTable(this, 'Time', textSort, true)" href="#">Time</a></th>
+			<th><a onclick="return sortTable(this, 'Kernel', textSort)" href="#">Kernel</a></th>
+			<th><a onclick="return sortTable(this, 'Commit', textSort)" href="#">Commit</a></th>
+			<th><a onclick="return sortTable(this, 'Syzkaller', textSort)" href="#">Syzkaller</a></th>
+			<th><a onclick="return sortTable(this, 'Config', textSort)" href="#">Config</a></th>
+			<th><a onclick="return sortTable(this, 'Log', textSort)" href="#">Log</a></th>
+			<th><a onclick="return sortTable(this, 'Report', reproSort)" href="#">Report</a></th>
+			<th><a onclick="return sortTable(this, 'Syz repro', reproSort)" href="#">Syz repro</a></th>
+			<th><a onclick="return sortTable(this, 'C repro', textSort)" href="#">C repro</a></th>
+			<th><a onclick="return sortTable(this, 'Maintainers', textSort)" href="#">Maintainers</a></th>
+		</tr>
+		</thead>
+		<tbody>
+		{{range $b := .}}
+		<tr>
+			<td class="manager">{{$b.Manager}}</td>
+			<td class="time">{{formatTime $b.Time}}</td>
+			<td class="kernel" title="{{$b.KernelAlias}}">{{$b.KernelAlias}}</td>
+			<td class="tag" title="{{$b.KernelCommit}}
+	{{formatTime $b.KernelCommitDate}}">{{link $b.KernelCommitLink (formatShortHash $b.KernelCommit)}}</td>
+			<td class="tag">{{link $b.SyzkallerCommitLink (formatShortHash $b.SyzkallerCommit)}}</td>
+			<td class="config">{{if $b.KernelConfigLink}}<a href="{{$b.KernelConfigLink}}">.config</a>{{end}}</td>
+			<td class="repro">{{if $b.LogLink}}<a href="{{$b.LogLink}}">log</a>{{end}}</td>
+			<td class="repro">{{if $b.ReportLink}}<a href="{{$b.ReportLink}}">report</a>{{end}}</td>
+			<td class="repro">{{if $b.ReproSyzLink}}<a href="{{$b.ReproSyzLink}}">syz</a>{{end}}</td>
+			<td class="repro">{{if $b.ReproCLink}}<a href="{{$b.ReproCLink}}">C</a>{{end}}</td>
+			{{if $b.HasMaintainers}}
```

Comment:
It does not make sense to not emit `<td>` for Maintainers column, if we emitted `<th>` above. It only produces half-consistent table. And if we will add another column after Maintainers, the table become broken. The current code also skips `<th>` for Maintainers.

It's also ugly to emit whole table contents, but not the `<table> `tag itself and no caption. Solution for all of this is to introduce a new object that will contain caption and HasMaintainers flag besides crash list. And then pass such object to this template.